cmd/ls.go

```diff
@@ -1,24 +1,39 @@
 package cmd
 
 import (
+	"context"
 	"fmt"
 	"sort"
+	"strings"
 
 	"github.com/urfave/cli"
 
 	"github.com/manifoldco/torus-cli/api"
+	"github.com/manifoldco/torus-cli/config"
 	"github.com/manifoldco/torus-cli/errs"
-	"github.com/manifoldco/torus-cli/identity"
 	"github.com/manifoldco/torus-cli/pathexp"
 )
 
+var targetMap = map[int]string{
```

Comment:
I don't think you gain anything using a map here. a slice with a bounds check during usage would be nicer IMO.